x-pack/elastic-agent/pkg/agent/program/testdata/journal_config-journalbeat.yml

```diff
@@ -1,15 +0,0 @@
-journalbeat:
```

Comment:
Argh, we supported it even in Elastic Agent? @nimarezainia We likely need to mention this somewhere also in Elastic Agent 8.0 breaking changes that `log/journal` was removed.

Reply:
I thought `journald` input was exposed under `log/journal`. Is that not correct?

Reply:
The problem is that it is correct. Under what path do we expose the filebeat journal input?